lib/Command/PreGenerate.php

```diff
@@ -96,21 +94,23 @@ public function __construct(string $appName,
 	protected function configure(): void {
 		$this
 			->setName('preview:pre-generate')
-			->setDescription('Pre generate only images that have been added or changed since the last regular run');
```

Comment:
Why change this description? Without the additional information, I find it pretty unclear what "pre-generating" is.

Reply:
This is an old pr request, this description probably got changed in master since this pr was originally created

Reply:
fixed